packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -43,13 +43,13 @@ watch(
 
 		if (modelId) {
 			let configs = await getModelConfigurationsForModel(modelId);
-			if (!configs[0].id) {
+			if (!configs[0]?.id) {
 				const model = await getModel(modelId);
 				if (model) await postAsConfiguredModel(model); // Create a model configuration if it does not exist
 				configs = await getModelConfigurationsForModel(modelId);
 			}
 			// Auto append output
-			if (configs[0].id) {
+			if (configs[0]?.id) {
```

Comment:
These logical checks are a bit janky, like, what makes getModelConfigurationsForModel returns the result such that the first-element IS the default config? 

Let's patch this logic up since we are not in a super hurry, e.g. if we need API support let's just add that.

Reply:
Sure, Ill take a look and maybe see returning model configs in order by oldest date (we can assume the oldest date is the default config since its the first created).

Reply:
@mwdchang I've updated the API to ensure that we get the configs in order of createdBy (oldest -> newest).  Let me know what you think.